src/tracing/options.ts

```diff
@@ -41,6 +41,11 @@ import {
 import { SplunkBatchSpanProcessor } from './SplunkBatchSpanProcessor';
 import { Resource } from '@opentelemetry/resources';
 
+const JaegerExporter = util.deprecate(
+  OriginalJaegerExporter,
+  'Jaeger exporter is deprecated and will be removed in 2.x'
```

Comment:
More detailed wording here: https://github.com/signalfx/gdi-specification/pull/191/files#diff-1a09c9dbbf6aa35c4524350e624d13e3f2f03221c21c1c75fde7e61144348452R229-R233

Reply:
Thanks! Changed the wording.